portal-web/test/functional/com/liferay/portalweb/paths/portlet/abtest/ABTest.path

```diff
@@ -68,6 +68,21 @@
 	<td>//*[contains(@class,'btn btn-primary') and contains(text(),'Save')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>VARIANT_DROPDOWN</td>
+	<td>//div[@class='autofit-col']//*[contains(@class, 'dropdown-toggle btn-monospaced btn btn-outline-borderless btn-outline-secondary')]</td>
```

Comment:
here you need to use all of these classes?

Reply:
yes, it was the only way i've found to find the path...